deployment/terraform/loadbalancing.tf

```diff
@@ -227,7 +219,7 @@ resource "aws_alb_listener_rule" "monitoring" {
 resource "aws_alb_target_group_attachment" "monitoring_a" {
   target_group_arn = "${aws_alb_target_group.monitoring.arn}"
   target_id        = "${aws_instance.nixops.id}"
-  port             = "3000"
+  port             = "80"
```

Comment:
Is this also the nginx port?

Reply:
oh good spot, I'll put it in a local var whatever it is

Reply:
Fixed, note that the `aws_alb_target_group` port is pretty much a default as that is the incoming port for the load balancer, i.e. 80 or 443 depending on who's dealing with SSL.

Reply:
Put that in a comment!